services/analyzer/mysql-table-constraints-getter.js

```diff
@@ -0,0 +1,54 @@
+function MysqlTableConstraintsGetter(databaseConnection) {
+  const queryInterface = databaseConnection.getQueryInterface();
+
+  this.convertToUniqueIndexArray = (constraints) => {
```

Comment:
A little suggestion for readability
```suggestion
  this.regroupUniqueConstraintIntoArray = (constraints) => {
```

Reply:
I don't find `regroup` clearer than `convert`:)